### DISCUSSION TITLE
sharing: normalize.min.css does not load

### DISCUSSION THREAD

**Discussion by maphew** [TriliumNext/Trilium#6013] [Ideas] [answered]
### Description

When visiting a publicly shared note from an anonymous browser there is this error in developer tools:

> Refused to apply style from 'https://.....com/assets/v0.90.12/libraries/normalize.min.css' because its MIME type ('application/json') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

### TriliumNext Version

v0.90.12

### What operating system are you using?

Other Linux

### What is your setup?

Server access only

### Operating System Version

Pika Pods container

**Comment by eliandoran** [comment]
@maphew , I cannot reproduce this issue. Are you sure you are not using a custom share template that might be affecting this?

See https://triliumnext.pikapod.net/share/fmFFWRBhf3Xm for reference.

![image](https://github.com/user-attachments/assets/8b2b0e75-1eb1-4537-a164-9691d06ffbfa)

**Comment by maphew** [comment]
I am using a custom share template, but assumed since the path is within v0.90.12 tree it wasn't coming from the custom theme. 

Ahh, found it, ` <link href="../<%= assetPath %>/libraries/normalize.min.css" rel="stylesheet">`, https://github.com/zerebos/trilium.rocks/blob/main/src/templates/page.ejs#L9.

**Comment by maphew** [comment]
I see normalize.min.css was pulled out of repo as part of https://github.com/TriliumNext/Notes/issues/224, which looks worthwhile. The thread is not clear on how to refer to or use the no-longer-hardcoded libraries now though. Can you offer insight on that?

**Comment by eliandoran** [comment, marked as answer]
> The thread is not clear on how to refer to or use the no-longer-hardcoded libraries now though. Can you offer insight on that?

There is no constraint on using any of the application's dependencies. However you have to make sure they are kept up to date, as there is no guarantee they will not change in time.

For your case, you can use: `../<%= assetPath %>/node_modules/normalize.css/normalize.css`
